index.js

```diff
@@ -64,6 +69,36 @@ module.exports = function (config) {
     if (viewBoxAttr) {
       $symbol.attr('viewBox', viewBoxAttr)
     }
+    
+    var attrs = $svg[0].attribs
+    for(var attrName in attrs){
+      if(attrName.match(/xmlns:.+/)){
+        var storedNs = namespaces[attrName],
+            attrNs = attrs[attrName]
+
+        if(storedNs != null){
```

Comment:
spaces around brackets, use strict comparison, perhaps this should be `storedNs  !== undefined`.